### PR TITLE
docs(ecosystem): update fastify-rabbitmq // add fastify-hl7 to ecosystem.md

### PR DESCRIPTION
**Pull request by Bugs5382**
* added in fastify-hl7
* updated description of fastify-rabbitmq

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [X] documentation is changed or added
- [X] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by Bugs5382** [comment]
Updated my own documentation for the ```fastify-rabbitmq``` plugin and added my newest plugin for  fastify: ```fastify-hl7```

File: docs/Guides/Ecosystem.md [hunk 1]
```diff
@@ -551,7 +556,7 @@ middlewares into Fastify plugins
   [qs](https://github.com/ljharb/qs).
 - [`fastify-rabbitmq`](https://github.com/Bugs5382/fastify-rabbitmq) Fastify
   RabbitMQ plugin that uses
-  [node-amqp-connection-manager](https://github.com/jwalton/node-amqp-connection-manager)
+  [node-rabbitmq-client](https://github.com/cody-greene/node-rabbitmq-client))
```
Comment (Uzlopak) [inline docs/Guides/Ecosystem.md:559]: Isnt the `)` too much?

```suggestion
  [node-rabbitmq-client](https://github.com/cody-greene/node-rabbitmq-client)
```

Comment (Bugs5382) [reply docs/Guides/Ecosystem.md]: Doh! and Done!

**Comment by Uzlopak** [comment]
Usually if we add a plugin to our ecosystem, we make a quick review to see if the code is on an acceptable level. When I see your code, then the default branch is basically empty :D. Did you forget to merge your develop branch into the main branch?
The code in the develop branch looks up to standard. Was just confused.

**Comment by Bugs5382** [comment]
> Usually if we add a plugin to our ecosystem, we make a quick review to see if the code is on an acceptable level. When I see your code, then the default branch is basically empty :D. Did you forget to merge your develop branch into the main branch? The code in the develop branch looks up to standard. Was just confused.

I usually "sync" your main into my fork. Create a branch. Then do a PR merged from that back to fastify/main. Not sure why you are seeing blank...

**Review by Bugs5382 — COMMENTED** [review]
LTGTM

**Review by Uzlopak — APPROVED** [review]
LGTM

**Comment by Uzlopak** [comment]
@Bugs5382 

The src folder in your default branch is empty.

**Comment by Bugs5382** [comment]
@Uzlopak  - Ah. Well, I do all my testing and develop in the develop branch (https://github.com/Bugs5382/fastify-hl7/tree/develop). I have published this on the development channel (https://www.npmjs.com/package/fastify-hl7/v/1.0.0-beta.2) overnight so I review it one more time in the morning when my eyes are fresh. :) But it is 100% ready to go, I beleive.

**Comment by Fdawgs** [comment]
Knew a HL7 plugin would finally turn up.
From reading the quick start it seems similar to Mirth Connect where you're spinning up TCP/IP HL7 listeners?

**Review by mcollina — CHANGES_REQUESTED** [review]
fastify-hl7 is mostly an empty github repo.

**Comment by Bugs5382** [comment]
Merged the code into the main branch. 👍  Released onto the NPM reg.

**Comment by Bugs5382** [comment]
> Knew a HL7 plugin would finally turn up. From reading the quick start it seems similar to Mirth Connect where you're spinning up TCP/IP HL7 listeners?

I am. 

It spin up TCP'MLLP inbound connections. It can also "connect" to a remote side. I built with with based off **node-rabbitmq-client** which has a built in auto reconnect for the client at least so that if the server goes off line (i.e. in a kubernetues network or just it restarting from the 'server' going down. Also used another plugin as the base, which hadn't been updated for two years to build HL7 messages as well in strong-typescript fashion. It uses two additional non-fastily plugins I wrote for the underlining technology. :)

**Review by Bugs5382 — COMMENTED** [review]
It's now in the main branch.

**Comment by Uzlopak** [comment]
@mcollina PTAL

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**